Adding verbose option to pass to planck

jeff303 opened the pull request.

jeff303 commented:
@mfikes , were you looking for some changes on this one?

mfikes commented:
@jeff303 Yes. If you can, please remove the change to the project version in `project.clj`

jeff303 commented:
> @jeff303 Yes. If you can, please remove the change to the project version in `project.clj`

My mistake; reverted that.  I also added another part of the condition so this will only be added for `planck` (since I haven't tried `lumo`, so I'm not sure if it would have the same option name and semantics).
